logger/unilog.go

```diff
@@ -208,6 +210,8 @@ func (it *independentTags) GetTags(metricName string) []tagPair {
 	if it == nil {
 		return []tagPair{}
 	}
+	it.Lock()
```

Comment:
Are we only using Lock/Unlock? If that's the case, does this need to be an RWMutex, or could we use a regular Mutex instead?